src/sql/src/session/vars/definitions.rs

```diff
@@ -2049,7 +2049,7 @@ feature_flags!(
     {
         name: enable_reduce_mfp_fusion,
         desc: "fusion of MFPs in reductions",
-        default: false,
+        default: true,
```

Comment:
@ggevay / @mgree

Reply:
This is fine to be on by default. (But I wouldn't completely remove it yet. There were no complaints about this so far, but it involves some tricky heuristics, so something might still come up.)